packages/account/src/Components/scrollbars-container/scrollbars-container.tsx

```diff
@@ -6,19 +6,27 @@ import { isMobile } from '@deriv/shared';
 type TScrollbarsContainer = {
     className?: string;
     scroll_offset?: string;
+    isFullHeight?: boolean;
 };
 
 export const ScrollbarsContainer = ({
     children,
     className,
     scroll_offset,
-}: React.PropsWithChildren<TScrollbarsContainer>) => (
-    <ThemedScrollbars is_bypassed={isMobile()} height={scroll_offset ? `calc(100% - ${scroll_offset})` : '100%'}>
-        <div
-            className={classNames('account__scrollbars_container', className)}
-            data-testid='dt_scrollbar_container_div'
+    isFullHeight = false,
+}: React.PropsWithChildren<TScrollbarsContainer>) => {
+    const height_unit = isFullHeight ? '100vh' : '100%';
+    return (
+        <ThemedScrollbars
+            is_bypassed={isMobile()}
```

Comment:
Is the themed scrollbar bypassed only for mobile? shall we make it !isDesktop 🙏

Reply:
True let me update this. 
BTW i did not add this prop it was already there 😁 

Thanks @utkarsha-deriv for highlighting

Reply:
Thank you 🙏